### ISSUE TITLE
Unable to find vive bindings

### ISSUE THREAD

**Issue by ivo1981** [DanielRBowen/UE4-VRHands-Template#4]
Hi, I'm pretty new to ue and found your project. 
I installed it and it ran. The problem i run in to is that i can't seem to find in which blueprint i can set the controller actions. So the hands basically do very little.
I tried manually binding buttons from steam, but this is depreciated. From what i understand we are supposed to use the openxr plugin and define input actions and then create a IMC to bind the actions to a controller.

So basically my question is where do I define the controller bindings so I can run the project and the controllers work as intended?
